### ISSUE TITLE
Add new item

### ISSUE THREAD

**Issue by dimka665** [tamtakoe/oi.select#33]
May be it's better to realize `selectize` new item adding flow instead of `prompt` and `autocomplete`?
After typing we can enter to select first item or up and enter to create new one.
First demo example: http://brianreavis.github.io/selectize.js/

**Comment by tamtakoe** [comment]
Good idea, but it is difficult for perfectly realisation because oi-select in contrast to selectize realizes round robin elements in dropdown. I make it in future.
